### PR TITLE
Mark some PPU opcodes as unimplemented

### PR DESCRIPTION
**Pull request by raven02**

### CODE REVIEW

File: rpcs3/Emu/Cell/PPUInterpreter.cpp [hunk 1]
```diff
@@ -150,12 +150,13 @@ const g_ppu_scale_table;
 
 bool ppu_interpreter::MFVSCR(PPUThread& ppu, ppu_opcode_t op)
 {
-	throw std::runtime_error("MFVSCR" HERE);
+	LOG_WARNING(PPU, "MFVSCR is unimplemented");
```
Comment (tambry) [inline rpcs3/Emu/Cell/PPUInterpreter.cpp]: I think an exception would make much more sense, as it's quite a serious error/problem.

Comment (raven02) [inline rpcs3/Emu/Cell/PPUInterpreter.cpp]: However it is indeed unimplemented , isn't?

Comment (tambry) [inline rpcs3/Emu/Cell/PPUInterpreter.cpp]: Yes. And unimplemented instruction is something that I would consider very serious and which should abort the execution, so it'd get reported and implemented properly.

**Comment by Nekotekina** [comment]
These aren't really unimplemented, they don't require implementation since they are No Ops, except MTVSCR when MFVSCR has been intentionally disabled. Is there any other reason to log them?

**Comment by Nekotekina** [comment]
And did someone encounter ECIWX instruction, by the way?

**Comment by raven02** [comment]
@Nekotekina .Then probably not worthwhile log it if no need to implement . BTW , so far we didn't encounter any ECIWX instruction yet.

**Comment by raven02** [comment]
Closing it .

**Pull request closed without merging**